### ISSUE TITLE
Q-input w/ validation always get cleared during composition when using Japanese IME on Android

### ISSUE THREAD

**Issue by xiaoxiao99** [quasarframework/quasar#13680]
### What happened?

This issue happens only if ALL the following conditions are true.
1. **Q-input** inside **Q-page** element, which is enclosed by **Q-page-container** then **Q-layout**.
2. **Q-input** has validation rules
3. Android phone
4. Screen in portrait mode
5. Japanese IME input SW keyboard

When all above are true, the keyboard inputs are always automatically cleared during composition, thus can not input any text.

### What did you expect to happen?

Expect Input box to show my input.

### Reproduction URL

https://codepen.io/xiaoxiao22/pen/jOZQQzL

### How to reproduce?

1. Open browser app on Android phone
2. Visit https://codepen.io/xiaoxiao22/pen/jOZQQzL
3. Switch input method to Japanese IME
4. Focus on input "w/o validation", type anything, no issue observed.
5. Focus on input "w/ validation", type something, the issue can be observed.

The codepen page may be less explanative because this is related to Q-layout while the layout is wrapped there.
So I create a Github repo to show it more clearly.
The code in the repo is the same as codepen page.

### Repo page:
https://github.com/xiaoxiao99/quasar2-input-issue

### Demo page:
https://xiaoxiao99.github.io/quasar2-input-issue/index.html

### Screenshot of demo page

https://user-images.githubusercontent.com/8103965/173372686-19b498c4-e674-43a1-9c69-709ba3c94664.mp4

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome, Android

### Quasar info output

```Shell
Quasar 2.7.1 running in UMD mode
(Same issue exists in CLI mode)
```

### Relevant log output

```Shell
N/A
```

### Additional context

This issue happens with Japanese IME but not other IMEs
One notable difference of Japanese IME is, its height grows by two lines when the inputs starts, as showed below.
This is not true for other IMEs, neither for Japanese IME in landscape screen.

![Img-image](https://user-images.githubusercontent.com/8103965/173383059-26d24925-385b-4949-99c7-36d939d0becf.png)

**Comment by xiaoxiao99** [comment]
> Hi @xiaoxiao99! 👋
> 
> It looks like you provided an invalid or unsupported reproduction URL. Do not use any service other than [Codepen](https://codepen.io), [jsFiddle](https://jsfiddle.net), [StackBlitz](https://stackblitz.com), [Codesandbox](https://codesandbox.io), and [GitHub](https://github.com). Make sure the URL you provided is correct and reachable. You can test it by visiting it in a private tab, another device, etc. Please **edit your original post above** and provide a valid reproduction URL as explained.
> 
> Without a proper reproduction, your issue will have to get closed.
> 
> Thank you for your collaboration. 👏

Updated reproduction URL

**Comment by pdanpdan** [comment]
Hello. Can you please test with the latest released version - there were some changes related to composition (not specific to this issue) and I want to know if the problem is still there?

**Comment by xiaoxiao99** [comment]
> Hello. Can you please test with the latest released version - there were some changes related to composition (not specific to this issue) and I want to know if the problem is still there?

Hello. Thank you for looking into this. Just tested with V2.7.3, it is still there. 
Updated the reproduction page a little so that user can know the Quasar version in action.

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.7.4
